Fix login redirections from any page (#123)
vtex-apps/login: CHANGELOG.md

and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]
### Changed

> - Fixes redirection after OAuth login when the user is in all the pages

Reviewer: ```suggestion
- Fix redirection after OAuth login.
```

> - Pass the returnUrl query when open login in mobile version

Reviewer: ```suggestion
- Pass the `returnUrl` query when the user opens login on mobile.
```
